modules/Emails/include/ListView/ListViewDataEmails.php

```diff
@@ -116,849 +132,590 @@ public function __construct()
         parent::__construct();
     }
 
+
     /**
-     * @inheritdoc
+     * @param User $currentUser
+     * @param Folder $folder
+     * @return InboundEmail
+     * @throws SuiteException
      */
-    public function getListViewData(
-        $seed,
-        $where,
-        $offset = -1,
-        $limit = -1,
-        $filter_fields = array(),
-        $params = array(),
-        $id_field = 'id',
-        $singleSelect = true,
-        $id = null
-    ) {
-        global $current_user;
-        global $sugar_config;
-        global $db;
-        global $mod_strings;
+    protected function getInboundEmail(User $currentUser, Folder $folder) {
```

Comment:
This is just a note for the pull request that type hinting with a callable is supported in PHP 5.4 and above. No action is required here.